### PR TITLE
NT-0039 - server with TypeScript

### PR DESCRIPTION
**Pull request by Foukz**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,35 +1,33 @@
 {
-    "env": {
```
Comment (mras2303) [inline .eslintrc]: Please, don't mess up file which you don't use

File: .gitignore [hunk 2]
```diff
@@ -8,6 +8,7 @@ jspm_packages
 # production
 build
 storybook-static
+server/build
```
Comment (mras2303) [inline .gitignore]: Previously we added `node_modules` to gitignore and it was working for every node_modules. Is this `build` works different?

**Comment by mras2303** [comment]
This pull request should require three approves.

@mras2303 
@damian0o 
@fhranis

File: .gitignore (hunk 2 above)
Comment (Foukz) [reply .gitignore]: No, that was my bad, sorry. After first creation of build I didn't get that low opacity effect like on other .gitignore files. But now everything seems to be ok.

File: .eslintrc (hunk 1 above)
Comment (Foukz) [reply .eslintrc]: I won't

**Pull request closed without merging**